settings.gradle

```diff
@@ -0,0 +1,12 @@
+/*
+ * This file was generated by the Gradle 'init' task.
+ *
+ * The settings file is used to specify which projects to include in your build.
+ * 
+ * Detailed information about configuring a multi-project build in Gradle can be found
+ * in the user guide at https://docs.gradle.org/4.8.1/userguide/multi_project_builds.html
+ */
+
+rootProject.name = 'htsjdk-next-beta'
+enableFeaturePreview('IMPROVED_POM_SUPPORT')
+
```

Comment:
We would like a multi-module project. See discusion in https://github.com/samtools/htsjdk/issues/896 - this requires changes in this file and probably the build.gradle

Reply:
We should decide if sub-modules are called in the form `htsjdk-*` or simply the package name (e.g., `htsjdk-core` vs. `core`)

Reply:
That's a good idea to set it up as a multimodule project... I'm not actually certain the best way to do it though.  Without some code to play with I don't think I'll get it right off the bat.  Do you have a suggestion for how to stub that?